apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -48,23 +50,38 @@ const nationalityData = [
   },
 ];
 
+const initialFormState = {
+  gender: 'male',
+  name: null,
+  lastName: null,
+  id: null,
+  nationality: null,
+  dateOfBirth: null,
+  bags: null,
+  passengerCount: 1,
+};
+
 type State = {
   ...PassengerCardType,
   lastName: ?string,
+  hasPrefilled: boolean,
 };
 
 class PassengerForm extends React.Component<Props, State> {
   state = {
-    gender: 'male',
-    name: null,
-    lastName: null,
-    id: null,
-    nationality: null,
-    dateOfBirth: null,
-    bags: null,
-    passengerCount: 1,
+    ...initialFormState,
+    hasPrefilled: false,
```

Comment:
```suggestion
    hasPrefilledState: false,
```
Will be better for understanding what's going on